assets/src/globals.js

```diff
@@ -20,13 +20,27 @@ const user = Object.freeze({
   helpURL: mylaGlobals.help_url
 })
 
-const palette = JSON.parse(JSON.stringify(defaultPalette))
+const sitePalette = JSON.parse(JSON.stringify(defaultPalette))
 // Don't update the palette if mylaGlobals.primary_ui_color is null or undefined
 if (mylaGlobals.primary_ui_color != null) {
-  palette.primary.main = mylaGlobals.primary_ui_color
+  sitePalette.primary.main = mylaGlobals.primary_ui_color
 }
 
-const siteTheme = createMuiTheme({ palette })
+// Variant mapping: https://material-ui.com/components/typography/#changing-the-semantic-element
+// This will make uses of h5 appear as h1 and h6 appear as h2, without changing styling.
+const componentSettings = {
+  MuiTypography: {
+    variantMapping: {
+      h5: 'h1',
+      h6: 'h2'
+    }
```

Comment:
Why is this change needed?

Reply:
If this has been implemented correctly, all uses of `h5` should appear as `h1`, and `h6` as `h2` in the page's HTML. According to Gonzalo, [Material UI](https://material-ui.com/components/typography/#changing-the-semantic-element), and [W3C](https://www.w3.org/WAI/tutorials/page-structure/headings/), it is important for accessibility that a page's headings start at `h1` and descend without skipping levels. Previously, we were using `h5` and `h6` as the preferred styles for headings, so I wanted to maintain those styles while making the heading numbers appear correctly in the HTML. Does that make sense?

Reply:
Indeed make sense, this is a kind global change for this inline version
`<Typography variant="h1" component="h5">`
This is reflected correctly in the UI and it doesn't have the same effect of `<h2>` header tag
![Screen Shot 2020-06-05 at 10 41 55 AM](https://user-images.githubusercontent.com/8579775/83889745-5e84bd80-a719-11ea-819c-6b5d23d7aac6.png)

![Screen Shot 2020-06-05 at 10 42 55 AM](https://user-images.githubusercontent.com/8579775/83889786-617fae00-a719-11ea-871a-05d3e7e0ccaa.png)